h5py/_hl/group.py

```diff
@@ -111,6 +113,32 @@ def create_dataset(self, name, shape=None, dtype=None, data=None, **kwds):
                 self[name] = dset
             return dset
 
+    def create_virtual_dataset(self,VMlist=[], fill_value=0x1):
```

Comment:
Why is the `VMlist` default an empty list? Are you aware of http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments? The commonly understood idiom is to set to `None, and create a new list inside the function, see the link.